app/team.go

```diff
@@ -252,7 +252,12 @@ func (a *App) sendTeamEvent(team *model.Team, event string) {
 	*sanitizedTeam = *team
 	sanitizedTeam.Sanitize()
 
-	message := model.NewWebSocketEvent(event, "", "", "", nil)
+	teamId := "" // no filtering by teamId by default
+	if event == model.WEBSOCKET_EVENT_UPDATE_TEAM {
```

Comment:
Currently the webapp `/select_team` view for joining and creating teams seems to update itself from websockets. How does this change affect it? Should this check account for public teams visible to all users on the server?